evil-visual-mark-mode.el

```diff
@@ -52,8 +52,7 @@
 
 This marker will normally come from the advised evil-set-marker
 function."
-  (let* ((marker-pos (marker-position marker)))
-    (make-overlay marker-pos (+ 2 marker-pos))))
+  (make-overlay marker marker))
```

Comment:
Shouldn't we make the marker of the size of 2 to properly render the backtick char with the letter marker?